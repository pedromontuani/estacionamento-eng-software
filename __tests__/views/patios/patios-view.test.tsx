import React from 'react';
import {rendererCreate} from '../../utils';

import PatiosView from '../../../src/views/patios/patios-view';

describe('Testing patios view', () => {
  test('should match snapshot', async () => {
    const tree = rendererCreate(<PatiosView />);
    expect(tree).toMatchSnapshot();
  });
});
