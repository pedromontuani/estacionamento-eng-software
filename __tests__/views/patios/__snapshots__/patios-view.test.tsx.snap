// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing patios view should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "flex": 1,
      "padding": 24,
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1976d2",
        "borderRadius": 20,
        "elevation": 5,
        "height": 40,
        "justifyContent": "center",
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 14,
          "fontWeight": "500",
        }
      }
    >
      Novo pátio
    </Text>
  </View>
  <RCTScrollView
    ListEmptyComponent={
      <Text>
        Nenhum pátio cadastrado no sistema
      </Text>
    }
    contentContainerStyle={
      Object {
        "padding": 24,
      }
    }
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "marginHorizontal": -24,
      }
    }
    viewabilityConfigCallbackPairs={Array []}
  >
    <View>
      <Text
        onLayout={[Function]}
      >
        Nenhum pátio cadastrado no sistema
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
